infra/scripts/replace-running-components.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+BASE_DIR=$(dirname $(readlink -f "$BASH_SOURCE"))
+VIC_DIR=$(dirname $(readlink -f $BASE_DIR/..))
+
+# Grab vSphere's thumbprint for calling vic-machine
+function get-thumbprint () {
+    govc about.cert | grep "SHA-1 Thumbprint" | awk '{print $NF}'
+}
+
+# Run the command given on the VCH instead of locally
+function on-vch() {
+    sshpass -ppassword ssh -oUserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no root@$VCH_IP -C $@ 2>/dev/null
```

Comment:
could we use the `vic-machine debug --authorized-key` option?
It's a path I don't think we test sufficiently but would also avoid hard coding dev passwords into the scripts.